Bottom: 343beca35626921995c7272aa8b209ebfe6a8701
Top:    73a3004f167bb30f4b2f8cee8239062b651b20f9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-11 19:10:01 +0000

Remove unused HistorySize field from the settings structure


---

diff --git a/src/global.h b/src/global.h
index 3794448..19f4454 100644
--- a/src/global.h
+++ b/src/global.h
@@ -668,8 +668,7 @@ typedef struct			/* some resources... */
     /* connections is done */
     AutoPlace;			/* flag which says we should force placement of the
 				   windows on startup */
-  int HistorySize,		/* FIXME? Used in hid/xaw code only. */
-    init_done;
+  int init_done;
 }
 SettingType, *SettingTypePtr;
